### PR TITLE
Dev design

### PR DESCRIPTION
**Pull request by FarazA22**
### Ticket

This pull request resolves [#187652840](https://www.pivotaltracker.com/story/show/187652840).

### Approach

I added a new color to our color palette and updated StoryBook (colors page)
Refactored the Action Bar Layout and used it in the Violations Component
Updated the primary button component ti enable the use op the new color and refactored the Violations Component to use the button component

## Code author checklist

- [x] I have performed a self-review of my code
- [x] I have created and/or updated relevant documentation (EX: [Engineering Reference/FAQ](https://docs.google.com/document/d/1X4iWSGmBZdHYZ0jGqkwTR3_yyyqI_TiiJptfc8pi9Po)), if necessary
- [x] I have not used any relative imports
- [x] I have checked for and removed instances of unused config from CMS
- [x] I have pruned any instances of unused code
- [x] I have not added any markdown to labels, titles and button text in config
- [x] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [x] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts` (see [CMS Additions in Engineering Reference/FAQ](https://docs.google.com/document/d/1X4iWSGmBZdHYZ0jGqkwTR3_yyyqI_TiiJptfc8pi9Po/edit#heading=h.fu5jdsrcqxbh))
- [x] I have updated relevant `.env` values in both `.env-template` and in Bitwarden

### CODE REVIEW

File: web/src/components/njwds-layout/ActionBarLayout.tsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ interface Props {
   children: ReactNode;
   stackOnLeft?: ReactNode;
   alignLeft?: boolean;
+  dontReverseInMobile?: boolean;
```
Comment (SirSamuelJoseph) [inline web/src/components/njwds-layout/ActionBarLayout.tsx:8]: Can we have this be the opposite? `reverseInMobile`? The double negative can be a bit harder to read

File: web/src/components/dashboard/ElevatorViolationsCard.tsx [hunk 2]
```diff
@@ -6,21 +8,30 @@ import { ReactElement } from "react";
 
 export const ElevatorViolationsCard = (): ReactElement => {
   const config = getMergedConfig();
+
+  const componentOnLeftSide = (
```
Comment (SirSamuelJoseph) [inline web/src/components/dashboard/ElevatorViolationsCard.tsx:12]: Can we rename this to `violationMessage`? makes sense for the prop to be `stackOnLeft` but within this component I think we can be more specific

**Review by SirSamuelJoseph — APPROVED** [review]
Two minor naming notes, great work otherwise, thanks for the design system teachings!

File: web/src/components/njwds-layout/ActionBarLayout.tsx (hunk 1 above)
Comment (FarazA22) [reply web/src/components/njwds-layout/ActionBarLayout.tsx]: added

File: web/src/components/dashboard/ElevatorViolationsCard.tsx (hunk 2 above)
Comment (FarazA22) [reply web/src/components/dashboard/ElevatorViolationsCard.tsx]: updatet

**Review by adidner — APPROVED** [review]
LGTM

**Pull request merged**